portal/src/locale-data/en.json

```diff
@@ -1455,12 +1455,21 @@
   "LoginMethodConfigurationScreen.password.description": "Configure the password policy for Password and Additional Password in MFA.",
   "LoginMethodConfigurationScreen.password.basic": "Basic",
   "LoginMethodConfigurationScreen.password.advanced": "Advanced",
+  "LoginMethodConfigurationScreen.lockout.title": "Account Lockout Policy",
+  "LoginMethodConfigurationScreen.lockout.threshold.title": "Lockout threshold",
+  "LoginMethodConfigurationScreen.lockout.threshold.description": "Configure the number of failed login attempts your users can make within a time limit before their accounts are locked.",
+  "LoginMethodConfigurationScreen.lockout.threshold.failedAttempts.title": "Failed Login Attempts",
+  "LoginMethodConfigurationScreen.lockout.threshold.failedAttempts.description": "This is the maximum failed attempts to login before the account is locked.",
+  "LoginMethodConfigurationScreen.lockout.threshold.resetAfter.title": "Reset failed attempts after (minutes)",
+  "LoginMethodConfigurationScreen.lockout.threshold.resetAfter.description": "The interval to reset failed attempts after the last failure.",
+  "LoginMethodConfigurationScreen.lockout.threshold.overall.description": "A user account will be locked after {attempts} failed login attempts. The failure attempts will be reset after {resetIntervalMins} minutes of the last failure. {resetIntervalHours, plural, =0 {} other {(i.e. {resetIntervalHours} hours)}}",
```

Comment:
It should be `{resetIntervalHours, plural, =0 {} one {(i.e. # hour)} other {(i.e. # hours)}`

And this is what is rendered:

`0`: `""`
`1`: `"(i.e. 1 hour)"`
other: `"(i.e. x hours)"`

The difference between `zero`, `=0` is that `zero` refers to the plural category, and `=0` is a simple numeric comparison. In English, the applicable plural categories are `one` and `other`, meaning that `englishPluralRule(number) => "one" | "other"`. So if we use `zero` here, it has no effect. But we also need to handle `one` here. The `#` sign allows us to refer to the number in question.

Reply:
@louischan-oursky I've changed this part in the later commit, because what we actually need is <1 but not 1 or 0, which is not what plural for.